paymentsheet/src/main/java/com/stripe/android/link/account/DefaultLinkAccountManager.kt

```diff
@@ -418,18 +445,23 @@ internal class DefaultLinkAccountManager @Inject constructor(
                 setAccount(
                     consumerSession = consumerSession,
                     publishableKey = null,
-                    displayablePaymentDetails = null
+                    displayablePaymentDetails = null,
+                    linkAuthIntentInfo = linkAccount.linkAuthIntentInfo, // Keep LAI info.
```

Comment:
the current pattern is a bit weird, but if you pass null, it should keep the existing cached value